Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.props

```diff
@@ -0,0 +1,33 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IsNuGetPackage>$(MSBuildThisFileDirectory.StartsWith($(NuGetPackageRoot)))</IsNuGetPackage>
+    <!-- Referencing 'lombiq.npm.targets' using all lowercase letters, because that's the way its spelled in GitHub
+         Actions which run on Linux, which uses a case-sensitive file system. On Windows this works flawlessly. -->
+    <LombiqNpmTargetsPath
+      Condition="'$(IsNuGetPackage)' == 'true'">$(NuGetPackageRoot.TrimEnd('/\'))/lombiq.npm.targets/1.0.1-alpha.5/build</LombiqNpmTargetsPath>
+    <LombiqNpmTargetsPath
+      Condition="'$(IsNuGetPackage)' != 'true'">$(MSBuildThisFileDirectory.TrimEnd('/\'))/../../lombiq.npm.targets</LombiqNpmTargetsPath>
```

Comment:
Can't this be done otherwise? Even if NPM Targets needs adjusting, or if we need a separate targets file too. This looks like a fragile implementation that we need to take special care of maintaining. We should rather get this working with how props files from NuGet are imported.

Reply:
Well, `.props` files from NuGet packages are only imported into projects from immediate references, not transitive ones. Without pulling in the `lombiq.npm.targets` here, they won't be imported at all.

What exactly is it that you deem fragile?

Reply:
`IncludeAssets` and `PrivateAssets` affect this. Can't they be used?

Instead of relying on NuGet to import everything we need, we're trying to look up the file in the package cache. This is working around the features NuGet provides for this scenario, and that shouldn't be necessary. Such things are alway risky, and here we also need to keep the version number up to date, something that we'll forgot a lot (since updating NuGet dependencies doesn't necessarily involve looking at the csproj file).

Reply:
I've never heard of `IncludeAssets` and `PrivateAssets,` so I'll go and change that 😁

Reply:
OK, so I've done some investigation and tests and I am a bit frustrated at this point because:
1. [This paragraph](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#:~:text=Note%20that%20because,that%20consume%20AppLogger.) reads to me like I should be able to provide `NPM Targets` _.props_ and _.targets_ files to consumer projects by specifying the following in `Lombiq.NodeJs.Extensions.csproj`: 
```xml
    <PackageReference Include="Lombiq.Npm.Targets" Version="1.0.1-alpha.6">
      <!-- We want to make NPM-Targets' .props and .targets files available to consumer projects. -->
      <IncludeAssets>all</IncludeAssets>
      <!-- We don't want to consume neither build not compile assets in NE itself. -->
      <ExcludeAssets>build;compile</ExcludeAssets>
      <!-- We don't want the NPM-Targets DLL in the consumer projects. -->
      <PrivateAssets>compile</PrivateAssets>
    </PackageReference>
```
2. But it simply does not work. I've tried a myriad of values, rebuilt the project, repacked the NuGet package (locally), installed the latest package - the NPM Targets build files are not included in the generated `.nuget.g.props` and `.nuget.g.targtes` files.

Any ideas?

Reply:
I suggest posting a question here: https://github.com/dotnet/msbuild/discussions or on Stackoverflow.

Reply:
OK, I've got the inclusion of .props and .targets to work - after adjusting the `.nuspec` file in the locally created package 😁 There seem to be a few bugs during its creation 🤔 

Now, I've pushed all of my changes and triggered the GitHub publish workflow, and now the `.nuspec` file in the NE package contains a `<repository>` node for NPM Targets instead of a `<dependency>` one. Do you know where this comes from?

![image](https://user-images.githubusercontent.com/164360/161637522-5d2d6c09-ddd3-4297-a8e0-02764ac0ea6a.png)

Reply:
I've filed [an issue](https://github.com/NuGet/Home/issues/11717) now that I've gathered more info on this. It seems quite impactful!

Reply:
Great, let's see the feedback. Can we override just this from a custom nuspec file as a workaround?

Reply:
I had the same idea, but since I've never worked with them, I don't know if it's going to work. I will try it out today.

Reply:
Looks like the _.nuspec_ file cannot override anything - it has to be used **in place of** the project file. See [the docs](https://docs.microsoft.com/en-us/nuget/reference/nuspec#project-type-compatibility).

Reply:
After trying for about 2h and ever new errors popping up, I gave up on tuning the `.nuspec` file by hand.

I have a working configuration now which include the `Npm.Targets` build files before those from `NodeJs.Extensions` 🎉